DOCS:  Add changelog entry for Field Lists changes. (#184)
open-contracting/kingfisher-summarize: docs/changelog.rst

Changelog
=========

2020-12-09
----------

> Changed
> ~~~~~~~
> 
> -  New command ``--field-lists`` that adds a ``field-list`` column to each summary table. The ``field-list`` column is an array of all fields in the data.
> 
> -  New columns in summary tables
> 
>    -  ``transaction`` JSONB of ``transaction`` object in ``contract_implementation_transactions_summary``
>    -  ``planning`` JSONB of ``planning`` object in planning_summary
> 

Reviewer: ```suggestion
Added
~~~~~

-  ``add`` command: ``--field-lists`` option: Adds a ``field_list`` column to all summary tables. The ``field-list`` column is an array of all fields in the data.
-  ``planning_summary`` table: A ``planning`` JSONB column for the planning object
-  ``contract_implementation_transactions_summary`` table:  A ``transaction`` JSONB column for the transaction object
```
